component/manage-service/src/main/java/com/wso2telco/dep/manageservice/resource/dao/ResponseFilterDAO.java

```diff
@@ -282,4 +284,14 @@ private boolean isInvalidResponseFilter(ResponseFilter responseFilter) {
         }
         return true;
     }
+
+    private boolean isSameOperation(String thisOp, String thatOp) {
+        String[] thisComponents = thisOp.split(" ");
+        String[] thatComponents = thatOp.split(" ");
```

Comment:
Better to keep meaningful names for variable names

Reply:
The reason here I used the words `this` and `that` is it is more elegant to use such words rather than numbered parameters like `o1` and `o2`. The parameters of this method are interchangeable and this method can be used to check the equality of any two operation strings in the `httpVerb+resource` format. Therefore if we use parameter names like 'reqOperation` and `dbOperation`, it will mislead the reader.